3-enrich/scala-common-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/enrichments/EnrichmentManager.scala

```diff
@@ -264,11 +243,39 @@ object EnrichmentManager {
       }
     }
 
-    // Finally anonymize the IP address
+    // Anonymize the IP address
     Option(event.user_ipaddress).map(ip => event.user_ipaddress = registry.getAnonIpEnrichment match {
       case Some(anon) => anon.anonymizeIp(ip)
       case None => ip
     })
+     
+    // Parse the useragent using user-agent-utils
+    val client = {
+      registry.getUserAgentUtilsEnrichment match {
+        case Some(uap) => {
+          Option(event.useragent) match {
+      	  case Some(ua) =>
+               val ca = uap.extractClientAttributes(ua)
```

Comment:
Indentation is slightly off. cases are not aligned. line up val ca ... with Option(event.useragent) indentation